lib/metadata_json_lint.rb

```diff
@@ -136,10 +142,26 @@ def validate_requirements!(requirements)
       if requirement['name'] == 'pe'
         warn :requirements, "The 'pe' requirement is no longer supported by the Forge."
       end
+
+      begin
+        VersionRequirement.new(requirement.fetch('version_requirement', ''))
+      rescue ArgumentError => e
+        # Raised when the version_requirement provided could not be parsed
+        error :dependencies, "Invalid 'version_requirement' field in metadata.json: #{e}"
+      end
+
+      validate_puppet_ver!(requirement) if options[:strict_puppet_version]
     end
   end
   module_function :validate_requirements!
 
+  def validate_puppet_ver!(range)
+    min_ver = range['version_requirement'].gsub(/[><=~^]/, '').split(' ')[0]
```

Comment:
Can we parse this using `VersionRequirement` rather than a regex? This will also fail with a nil error if the key is not set.